installbuilder/Makefile

```diff
@@ -14,7 +14,7 @@ ifeq ($(UNAME_P),x86_64)
 else
  PF_ARCH := x86
 endif
-CONFIG_VERSION := 1.1.1
+CONFIG_VERSION := $(DSC_BUILDVERSION_MAJOR).$(DSC_BUILDVERSION_MINOR).$(DSC_BUILDVERSION_PATCH)
```

Comment:
This is so that dsc dynamically picks up the version number from the Build_powershell_DSC_for_linux repo